vectorsearch_embedding/workload.py

```diff
@@ -0,0 +1,214 @@
+import random
+import os
+import json
+import asyncio
+import time
+import traceback
+from pathlib import Path
+
+from osbenchmark.workload.loader import Downloader
+from osbenchmark.workload.loader import Decompressor
+from osbenchmark.workload.loader import Decompressor
+from osbenchmark.worker_coordinator.runner import Runner
+from osbenchmark.worker_coordinator.runner import time_func
+from osbenchmark.exceptions import BenchmarkError
+
+script_dir = os.path.dirname(os.path.realpath(__file__))
+
+class DeleteIngestPipeline(Runner):
```

Comment:
is this operation not present in OSB? This is a pretty generic operation we should move this Operation of OSB.

Reply:
Moved this operation to OSB.